app/styles/components/_horizontal-nav.scss

```diff
@@ -29,11 +29,16 @@
   background-color: $denim-blue;
   z-index: 100;
 
+  .horizontal-nav__nav {
+    margin-top: 10px;
+  }
+
   .donate-button-link {
     position: relative;
     padding-left: 30px;
     padding-top: 10px;
     padding-right: 0;
+    display: none; //temporary measure: redesign in progress
```

Comment:
is this still needed?

Reply:
If it's coming back (or some iteration of it) in the future, is it okay to keep it in there and just hide it for now?

Reply:
why is this is `_horizontal-nav.scss`? it might be better located in the header scss fieldworker, since that's where the donate button is placed, right?